pkg/controller/factory/base_controller.go

```diff
@@ -120,35 +121,37 @@ func (c *baseController) Run(ctx context.Context, workers int) {
 	// if scheduled run is requested, run the cron scheduler
 	if c.resyncSchedules != nil {
 		scheduler := cron.New()
+		scheduledJob := newScheduledJob(c.Name(), c.syncContext.Queue())
 		for _, s := range c.resyncSchedules {
-			scheduler.Schedule(s, newScheduledJob(c.name, c.syncContext.Queue()))
+			schedule, err := cron.ParseStandard(s)
+			if err != nil {
+				panic(fmt.Errorf("invalid controller schedule %q: %v", s, err))
```

Comment:
why panic?

Reply:
it is programmer error (you specified wrong schedule format), this is not retriable and controller should panic